Package.swift

```diff
@@ -15,16 +15,12 @@ let package = Package(
     targets: [
         .target(
             name: "Nimble", 
-            dependencies: {
-                #if os(macOS)
-                return [
-                    "CwlPreconditionTesting",
-                    .product(name: "CwlPosixPreconditionTesting", package: "CwlPreconditionTesting")
-                ]
-                #else
-                return []
-                #endif
-            }()
+            dependencies: [
+                .product(name: "CwlPreconditionTesting", package: "CwlPreconditionTesting",
+                         condition: .when(platforms: [.macOS, .iOS, .tvOS])),
+                .product(name: "CwlPosixPreconditionTesting", package: "CwlPreconditionTesting",
+                         condition: .when(platforms: [.macOS, .iOS, .tvOS]))
```

Comment:
```suggestion
                .product(name: "CwlPreconditionTesting", package: "CwlPreconditionTesting",
                         condition: .when(platforms: [.macOS, .iOS])),
                .product(name: "CwlPosixPreconditionTesting", package: "CwlPreconditionTesting",
                         condition: .when(platforms: [.tvOS]))
```

CwlPosixPreconditionTesting is only for `.tvOS`.